### DISCUSSION TITLE
Update 2020-11-09

### DISCUSSION THREAD

**Discussion by frol** [near/near-explorer#470] [Show and tell]
Our previous update is here: https://github.com/near/near-explorer/discussions/463

Find our Objectives and Key Results for 2020 Q4: https://github.com/near/near-explorer/discussions/467

# Summary by Focus

### Explorer for Non-tech Users

* Explorer UI updates are on their way, @icerove is working on #466 to bring the new Dashboard and overall redesign from #405

### Explorer for Contract Developers

* Indexer for Explorer is getting its final shape squashing some bugs (https://github.com/near/near-indexer-for-explorer/issues/32, https://github.com/near/near-indexer-for-explorer/issues/38) and bringing consistency in tables/columns naming and performance through Postgres indices (https://github.com/near/near-indexer-for-explorer/issues/37, https://github.com/near/near-indexer-for-explorer/issues/41). Kudos to @khorolets 

### Explorer for Validators

* The new design for Explorer Dashboard for Validators is getting some love from @corwinharrell (see his detailed notes in his comment below)

**Comment by khorolets** [comment]
Still working on stabilisation and some polishing on the database design side. Adding indices, renaming some columns to make them more understandable. Also we've discovered that Indexer for Explorer skips actions from some transactions (https://github.com/near/near-indexer-for-explorer/issues/42), so the actions are not saved into database, I'm investigating this to create a fix. 
After I fix the above I'm going to address https://github.com/near/near-indexer-for-explorer/issues/33 and find a way to introduce idempotent updated/inserts of access keys.

**Comment by heycorwin** [comment]
### 1. Validator Dashboard
Progress on this last week was slow with everything going on between explorer, wallet, DevX research, and rainbow bridge. I'm just about ready to share the first iteration. Looking to get feedback on the new design starting Tuesday, Nov 10.

### 2. App-wide design updates
In addition to the validator dashboard, it's going to be important to ensure that the design across the explorer is consistent with the upcoming changes happening as a part of the dashboard refresh. As @icerove gets farther along in the development for the new dashboard, I'll be providing some updates to the other views of the explorer to make sure consistency is maintained across the board, particularly with things like typography and other simple/functional components.

Before this occurs, I'd like to see what kind of impact the changes to things like typography and the new navigation have had on the explorer. @icerove if you could share some screenshots or a screen recording of those changes, that would be great in helping me to decide what other parts of the app might need some small tweaks. We shouldn't ship any changes from the dashboard refresh until we feel like confident that the design changes applied site-wide are consistent, and that it doesn't feel like the dashboard is too divergent from the other views.

### 3. Design Resourcing Conflicts w/ Rainbow Bridge Front-End
There is a critical need for design resources for designing the UX and UI for the new Rainbow Bridge front-end. The goal of the team working on this is to release an MVP by the end of the month. Over the next week, I will need to de-prioritize the explorer in order to complete design for the first version of the rainbow bridge front-end.

### 4. Hiring for Design
Before the re-org, there was some discussion about bringing on a second product designer, considering how things were ramping up after our launch. That hire was de-prioritized until we felt like the need to fill that role was more critical. Now that we've shipped mainnet as well as Phase II, it has become clear that we would benefit from additional product design help. Designing and Facilitating user research for Wallet / Explorer / DevX / and Bridge is too much for one person to handle, and it's feeling like Explorer often gets the leftovers of design resources since those other projects are often considered higher priority. I'm in the process of writing a new job req for a second product design hire, so that we can better distribute the design workload, create more opportunities for collaboration within the design team, and to ensure that the resourcing available for projects like the Explorer are more consistent.
